l10n_it_ddt/tests/test_ddt.py

```diff
@@ -74,6 +74,23 @@ def _create_sale_order(self):
             })
         return order
 
+    def _create_order_ddt(self, partner):
+        order2 = self._create_sale_order()
+        order2.partner_id = partner
+        order2.onchange_partner_id()
+        self.product3.ddt_invoice_exclude = False
+        self.product3.invoice_policy = 'order'
+        self._create_sale_order_line(order2, self.product1)
+        self._create_sale_order_line(order2, self.product3)
+        order2.create_ddt = True
+        order2.action_confirm()
+        order2.picking_ids.action_assign()
+        ddt2 = order2.ddt_ids[0]
+        ddt2.action_put_in_pack()
+        ddt2.action_done()
+        ddt2.to_be_invoiced = True
+        return ddt2, order2
```

Comment:
c'è un motivo particolare per cui hai usato `ddt2` e `order2`? Potresti rinominarli in `ddt` e `order`?

Reply:
Ffatto